backend/src/apiserver/visualization/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
For future reference can you drop a comment in the pr post on how you re-generated this file, it will be useful in the future for the next person that performs a similar pr for a newer version

Reply:
Each requirements.txt generated from pip-compile includes a comment with the command that was used to generate it. By checking the content of the file above, you'll find the following:
```
#
# This file is autogenerated by pip-compile with Python 3.9
# by the following command:
#
#    pip-compile --output-file=- --resolver=backtracking -
#
```